Code/Tools/FBuild/FBuildCore/Graph/ObjectNode.cpp

```diff
@@ -552,6 +552,14 @@ Node::BuildResult ObjectNode::DoBuildWithPreProcessor2( Job * job, bool useDeopt
     Args fullArgs;
     AStackString<> tmpDirectoryName;
     AStackString<> tmpFileName;
+
```

Comment:
Forcing the use of the preprocessor will cause the build to fail for various things (some things are completely incompatible with the preprocessor)